packages/evaluatorq-py/src/evaluatorq/evaluatorq.py

```diff
@@ -0,0 +1,120 @@
+import asyncio
+import os
+from collections.abc import Awaitable, Sequence
+from datetime import UTC, datetime
+from typing import cast
+
+from .fetch_data import fetch_dataset_as_datapoints, setup_orq_client
+from .processings import process_data_point
+from .progress import Phase, ProgressService, with_progress
+from .send_results import send_results_to_orq
+from .table_display import display_results_table
+from .types import DataPoint, EvaluatorParams, EvaluatorqResult
+
+
+async def evaluatorq(name: str, params: EvaluatorParams) -> EvaluatorqResult:
+    """
+    Run an evaluation with the given parameters.
+
+    Args:
+        name: Name of the evaluation run
+        params: Evaluation parameters including data, jobs, evaluators, etc.
+
+    Returns:
+        List of DataPointResult objects
+    """
+    # Destructure with .get() for defaults
+    data = params["data"]
+    evaluators = params.get("evaluators", [])
+    jobs = params["jobs"]
+    parallelism = params.get("parallelism", 1)
+    print_results = params.get("print", True)
+    description = params.get("description")
+
+    orq_api_key = os.environ.get("ORQ_API_KEY")
+
+    start_time = datetime.now(UTC)
```

Comment:
We can get the local time with `now_tz = datetime.now().astimezone()`, rather than UTC, which is something I would prefer at least. Any reason we prefer UTC?